python/hail/representation/pedigree.py

```diff
@@ -0,0 +1,321 @@
+from hail.java import *
+
+
+class Trio(object):
+    """Class containing information about nuclear family relatedness and sex.
+
+    :param str proband: sample ID of proband
+
+    :param father: sample ID of proband
+    :type father: str or None
+
+    :param mother: sample ID of proband
+    :type mother: str or None
+
+    :param sex: sex of proband: 1 for male, 2 for female
```

Comment:
I think is_female rather than sex with a fixed encoding is clearer.  (Or enums, but those require a library.)